extensions/cache/runtime/pom.xml

```diff
@@ -23,6 +23,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-caffeine</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-mutiny</artifactId>
+        </dependency>
```

Comment:
Is it possible to have this a `provided` dependency? 
It's not a big deal, but if we could avoid the hard dependency, it would be nice

Reply:
It's the first time I hear about a provided dependency between Quarkus extensions. What kind of effect would that have? I'm very familiar with that scope in apps deployed on EAP but not so much in Quarkus.

Also, if #8631 ends up being merged, I think the hard dependency will become necessary.

Reply:
Well, this is just a dependency of the runtime module, so it's not a full blown extension on extension optional dependency, is it?

But in any case it's not a big deal and as you say, with  #8631 the point will no longer be valid. So let's just leave it as it is